DS-019 Pixhawk versions and revisions.md

```diff
@@ -150,7 +150,7 @@ HWTYPE = {v5/6x}{VER}{REV}
 | 0x003 | resistors | NXP T1 PHY |
 | 0x004 | resistors | HB CM4 |
 | 0x005 | resistors | HB mini |
-| 0x006 | resistors | RESERVED DRONECODE |
+| 0x006 | resistors | ZeroOne X6 |
```

Comment:
This is already reserved you will need to use and EEPROM code point

Reply:
I want to determine the two resistance values on the base board, which value of base board versions can I use?

Reply:
Does this even apply if they are using their own build target? @davids5

Reply:
If we use EEPROM code, does it mean that the resistors on the baseboard and FMU board should use code 0x007 (46.4K (R1) and 84.5K for R2)?

Reply:
The FMUM has an id: HW FMUM ID: 0xXXX
The Base has an id: HW BASE ID: 0xXXX
Since you are adding board support for your FMUM you can choose any  ID you want for the FMUM - Rev 0 is nop pop of resistors. The code that runs on you FMUM will had to deal with any FMUM - Rev IDs you ship. This my be sensor changes etc. 

In contrast: If you included your FMUM board support in the px4_fmu-v6x board, you would use EEPROM code point resistors. Dronecode would give you a tool and an FMUM_ID to program on the board in manufacturing.

If you base board is different: Adds some feature not supported on other base board. (Your BASE is not 100% compatible with any of the already defined ID)  you would use EEPROM code point resistors. Dronecode would give you a tool and an BASE_ID to program on the board in manufacturing.

In contrast: If the BASE is exactly the same (electrically and connectors ) as a `Auterion v5x base board` or `Holybro base Board' you can use Config 0 (nop pop)

Reply:
The sensors on our FMU and IMU include: 2 ICM45686, 1 BMI088, 2 ICP20100, and 1 RM3100, which we are ready to use Rev 0(nop pop of resistors). 
Then the base board has some functional updates, such as PWM output 3.3V and 5V switching, and we are ready to use EEPROM code point resistors. 
Do you think that's okay?

Reply:
https://github.com/PX4/PX4-Autopilot/pull/23477
In this link, after I updated the pab_manifest.c, I encountered an error when uploading it to PR. Could you please help me find out the reason?

Reply:
> [PX4/PX4-Autopilot#23477](https://github.com/PX4/PX4-Autopilot/pull/23477) In this link, after I updated the pab_manifest.c, I encountered an error when uploading it to PR. Could you please help me find out the reason?

I have modified it according to your suggestion, please help to review it.